recipes/diligent-core/all/patches/0002-use_conan_dependencies.patch

```diff
@@ -0,0 +1,142 @@
+diff --git a/Graphics/GraphicsEngineVulkan/CMakeLists.txt b/Graphics/GraphicsEngineVulkan/CMakeLists.txt
+index 387b3b019..e760513de 100644
+--- a/Graphics/GraphicsEngineVulkan/CMakeLists.txt
++++ b/Graphics/GraphicsEngineVulkan/CMakeLists.txt
+@@ -170,7 +170,7 @@ set(PRIVATE_DEPENDENCIES
+ if (${DILIGENT_NO_HLSL})
+     message("HLSL support is disabled. Vulkan backend will not be able to consume SPIRV bytecode generated from HLSL.")
+ else()
+-    list(APPEND PRIVATE_DEPENDENCIES SPIRV-Tools-opt)
++    list(APPEND PRIVATE_DEPENDENCIES CONAN_PKG::spirv-tools)
+ endif()
+ 
+ # Use DirectX shader compiler for SPIRV.
+diff --git a/Graphics/ShaderTools/CMakeLists.txt b/Graphics/ShaderTools/CMakeLists.txt
+index 00e634263..a7519f23e 100644
+--- a/Graphics/ShaderTools/CMakeLists.txt
++++ b/Graphics/ShaderTools/CMakeLists.txt
+@@ -133,26 +133,26 @@ endif()
+ if(ENABLE_SPIRV)
+     target_link_libraries(Diligent-ShaderTools
+     PRIVATE
+-        spirv-cross-core
++        CONAN_PKG::spirv-cross
```

Comment:
Should we avoid the deprecated namespace?

Reply:
yes, call find_package(), these targets exist, some parts of this patch are not needed.